AnnoyancesFilter/sections/cookies_whitelist.txt

```diff
@@ -2,6 +2,11 @@
 !######### White list. Fixing filtering errors #########
 !#######################################################
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80069
+!#if (adguard_app_ios || adguard_ext_safari || adguard_ext_android_cb)
+greenweez.com#@##staticBackdrop
```

Comment:
No need for whitelist. Just change existing rule `greenweez.com###staticBackdrop` to `greenweez.com#$##staticBackdrop { display: none !important; }`

Reply:
We are both right in different perspective. I thought in a different way and you thought in a different way. We should different thoughts ;)

Reply:
Yours is one more rule.

Reply:
@BlazDT Fixed. Please take a look.